src/core/utils/nameOf.ts

```diff
@@ -1,19 +1,12 @@
-// const nameOf = <T>(name: Extract<keyof T, string>): string => name;
```

Comment:
previously this nameOf function had another purpose, it was only used for the urlParams, I have moved this code there, and implemented a new `nameOf` util, which is more similar to the c# nameOf util, it's useful for forms, see the comment in code